src/main/java/backend/IssueMetadata.java

```diff
@@ -54,7 +55,7 @@ public IssueMetadata invalidate() {
      * is filled in later.
      */
     public static IssueMetadata intermediate(List<TurboIssueEvent> events, List<Comment> comments,
-                                             String eventsETag, String commentsETag) {
+            String eventsETag, String commentsETag) {
```

Comment:
Can we go back to previous indention which is more readable? Is it not allowed by checkstyle?